mne/viz/topomap.py

```diff
@@ -76,6 +76,7 @@
 )
 
 _fnirs_types = ("hbo", "hbr", "fnirs_cw_amplitude", "fnirs_od")
+_opm_coils = (8002,)
```

Comment:
Don't use the `int` but rather the corresponding `FIFF.FIFFV_` instead. And there should be more entries, no?